docs/source/torch.compiler_aot_inductor.rst

```diff
@@ -211,6 +211,7 @@ Below are some useful tools for debugging AOT Inductor.
 
    logging
    torch.compiler_aot_inductor_minifier
+   torch.compiler_inductor_provenance
```

Comment:
This is not just an AOTInductor debugging tool, right? It is also a torch.compile debugging tool